src/commands/scanner/run.ts

```diff
@@ -110,6 +79,14 @@ export default class Run extends ScannerCommand {
 			description: messages.getMessage('flags.tsconfigDescription'),
 			longDescription: messages.getMessage('flags.tsconfigDescriptionLong')
 		}),
+		eslintconfig: flags.string({
+			description: messages.getMessage('flags.eslintConfig'),
+			longDescription: messages.getMessage('flags.eslintConfigLong')
+		}),
+		pmdconfig: flags.string({
+			description: messages.getMessage('flags.pmdConfig'),
+			longDescription: messages.getMessage('flags.pmdConfigLong')
+		}),
 		// TODO: This flag was implemented for W-7791882, and it's suboptimal. It leaks the abstraction and pollutes the command.
```

Comment:
Question: Does the work you're doing help move us towards getting rid of the `--env` flag? It's fine if not, I'm just curious.

Reply:
I believe yes but haven't tested this yet. User needs to have the config in their eslintrc.json and also keep all the dependencies in the directory they are running the scanner from.